app/__tests__/EmailJs.test.js

```diff
@@ -0,0 +1,16 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import React from "react";
+import { MemoryRouter } from "react-router-dom";
+import EmailJs from "../components/contacts/Email";
+
+test("Contacts renders correctly with a form", () => {
+  const { getByText } = render(
+    <MemoryRouter>
+      <EmailJs />
+    </MemoryRouter>
+  );
+
+  const form = document.createElement("form");
+  expect(form).not.toBeNull();
```

Comment:
you are creating a form in the test, not in the component, this is not needed

Reply:
correction done!